content/css/concepts/filter-functions/terms/opacity/opacity.md

```diff
@@ -38,3 +38,5 @@ Set the opacity of our element to `50%`:
   filter: opacity(50%);
 }
 ```
+
+![css-filterfunction-opacity-example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunction-opacity-example.png)
```

Comment:
For the output itself, other entries show the original image first followed by the image with filter to the right. For the sake of uniformity, please reorder those images.